### PR TITLE
small helperization to DRY common rust

### PR DESCRIPTION
**Pull request by zancas**
The common Rust base is wet!  This DRYs it a bit.

### CODE REVIEW

File: rust/lib/src/lib.rs [hunk 1]
```diff
@@ -51,19 +33,43 @@ pub fn init_new(server_uri: String, data_dir: String) -> String {
...
 
-pub fn init_from_seed(server_uri: String, seed: String, birthday: u64, data_dir: String) -> String {
-    let server = construct_server_uri(Some(server_uri));
-
-    let (mut config, _latest_block_height) = match zingolib::load_clientconfig(server, None) {
+    let (mut config, latest_block_height) = match zingolib::load_clientconfig(server, None) {
         Ok((c, h)) => (c, h),
         Err(e) => {
-            return format!("Error: {}", e);
+            return Err(format!("Config load Error: {}", e));
```
Comment (juanky201271) [inline rust/lib/src/lib.rs]: In zingo-mobile code base exists a convention about how to catch errors in the responses of RPCModule... In most of the places the App is waiting an error when starts with "Error:", can you change this line to: "Error Config load......"?????? Thanks.

**Review by juanky201271 — CHANGES_REQUESTED** [review]

File: rust/lib/src/lib.rs (hunk 1 above)
Comment (zancas) [reply rust/lib/src/lib.rs]: Like:   `"Error: Config load:..."` ?

Comment (zancas) [reply rust/lib/src/lib.rs]: OK, I pushed an update.  How does that look?

Comment (juanky201271) [reply rust/lib/src/lib.rs]: Yeap... thanks.

**Review by juanky201271 — DISMISSED** [review]

**Review by juanky201271 — APPROVED** [review]
good work.

**Pull request merged**